src/runner/index.js

```diff
@@ -33,17 +32,15 @@ export default class Runner extends EventEmitter {
             }
 
             bc.close();
-            bc.once('closed', resolve);
+
+            bc.once('disconnected', resolve);
         });
     }
 
     static async _freeBrowserConnection (bc, errorHandler) {
         bc.removeListener('error', errorHandler);
 
-        // NOTE: we should close local connections and
-        // related browsers once we've done
-        if (bc instanceof LocalBrowserConnection)
-            await Runner._waitForLocalBrowserClose(bc);
+        await Runner._waitForBrowserDisconnected(bc);
```

Comment:
We can't close non-local connections, since they maybe shared with the other runner.